### PR TITLE
[Rados] Automation coverage for bug fixes: 2011756 & 2229651

### PR DESCRIPTION
**Pull request by harshkumarRH**
Automation coverage for the following bug fixes:
- [Bug 2229651](https://bugzilla.redhat.com/show_bug.cgi?id=2229651) - [EC 2+2@4] Observing Slow OSD heartbeats on front and back post Host down and reboot
- [Bug 2011756](https://bugzilla.redhat.com/show_bug.cgi?id=2011756) - Configuration info is not showing for RGW daemon ([edit](https://bugzilla.redhat.com/show_bug.cgi?id=2011756#))

Jira trackers:
[RHCEPHQE-14407](https://issues.redhat.com/browse/RHCEPHQE-14407)
[RHCEPHQE-14392](https://issues.redhat.com/browse/RHCEPHQE-14392)

Pass logs-
Reef: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-97JD3E

Signed-off-by: Harsh Kumar <hakumar@redhat.com>

### CODE REVIEW

File: tests/rados/test_bug_fixes.py [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Module to verify scenarios related to fixes provided for generic RADOS BZs
+    which do not fit in any other existing segregation
+
+    Currently, covers the following tests:
+    - CEPH-: Verify ceph config show and get for all daemons
```
Comment (neha-gangadhar) [inline tests/rados/test_bug_fixes.py:24]: [Nit] We can add the polarion IDs here.

File: tests/rados/test_bug_fixes.py [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+                            daemon=daemon, id=rand_orch_ps["daemon_id"]
+                        )
+
+                    log.info(
+                        f"ceph config get output for the daemon {rand_orch_ps['daemon_name']}: "
+                        f"\n\n {config_get_op}"
+                    )
+                    log.info(
+                        f"ceph config show output for the daemon {rand_orch_ps['daemon_name']}: "
+                        f"\n\n {config_show_op}"
```
Comment (neha-gangadhar) [inline tests/rados/test_bug_fixes.py:112]: Is there a need to validate if both outputs are producing same results? Or have we seen they do not match in some cases?

File: tests/rados/test_bug_fixes.py [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+            assert rados_obj.add_network_delay_on_host(
+                hostname=rand_host, delay="1100ms", set_delay=True
+            )
+
+            # smart wait for 120 secs to check Slow OSD heartbeat warning
+            timeout_time = datetime.datetime.now() + datetime.timedelta(seconds=120)
+            while datetime.datetime.now() < timeout_time:
+                health_detail, _ = cephadm.shell(args=["ceph health detail"])
+                log.info(f"Health warning: \n {health_detail}")
+                if "Slow OSD heartbeats on back" not in health_detail:
```
Comment (neha-gangadhar) [inline tests/rados/test_bug_fixes.py:181]: please check the logic here. Some confusion with if/else on the warning generation.

**Review by pdhiran — APPROVED** [review]
Looks good.

**Pull request merged**